components/UnitWidget.vue

```diff
@@ -60,6 +31,32 @@ export default {
     },
   },
   computed: {
+    symbol() {
+      let symbol = ''
+      let space = ''
+
+      switch (this.unitType) {
+        case 'temp':
+          symbol = this.units == 'metric' ? '&deg;C' : '&deg;F'
+          break
+        case 'mm_in':
+          symbol = this.units == 'metric' ? '&#x339C;' : 'in'
+          space = '&#8239;'
+          break
+        case 'm_in':
+          symbol = this.units == 'metric' ? 'm' : 'in'
+          space = '&#8239;'
+          break
+        case 'dd':
+          symbol =
+            this.units == 'metric' ? '&deg;F&sdot;days' : '&deg;C&sdot;days'
```

Comment:
Looks like this ternary statement is backwards. It's showing °C•days when the report is set to imperial units:

![image](https://user-images.githubusercontent.com/2566292/199316445-f7bf6d87-eec3-42db-bc79-a8afbd3c10aa.png)